src/locales/en/ministry.json

```diff
@@ -122,6 +122,7 @@
   "tr_pioneers": "Pioneers",
   "tr_reminderPublisherReport": "Submit your monthly field service report",
   "tr_reminderPublisherReportDesc": "It should be submitted to the secretary no later than the 6th day of the month.",
-  "tr_reminderBranchReport": "The Congregation Field Service and Meeting Attendance (S-1) report hasn’t been submitted yet",
-  "tr_reminderBranchReportDesc": "It should be submitted to the branch office no later than the 20th day of the month."
+  "tr_reminderBranchReport": "The Congregation Field Service and Meeting Attendance (S-1) report is not yet ready",
```

Comment:
@FussuChalice: one last change please: could you update your local file in this line? It has been changed but it seems like yours is still the old one and this will overwrite the last update if not changed. Thanks.